modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -0,0 +1,402 @@
+= Anypoint Runtime Fabric Installation Prerequisites
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: install-port-reqs.adoc, install-sys-reqs.adoc
+
+Review the following requirements with your operations and IT teams to ensure that all prerequisites are met before installing Anypoint Runtime Fabric. The Runtime Fabric installation process fails if the minimum requirements are not met.
+
+If needed, contact your MuleSoft representative for assistance.
+ 
+See xref:architecture.adoc[Runtime Fabric Architecture] for a description of development and production configurations.
+
+== General Knowledge
```

Comment:
Can you come up with a more precise title? I think even User Knowledge or background or experience or similar is more precise, as indicated in my previous edit of this section.